qos/qos_flow.go

```diff
@@ -382,3 +382,15 @@ func (d *QosFlowDescriptionsAuthorized) AddDefaultQosFlowDescription(sessRule *m
 func (upd *QosFlowsUpdate) GetAddQosFlowUpdate() map[string]*models.QosData {
 	return upd.add
 }
+
+func GetDefaultQoSDataFromPolicyDecision(smPolicyDecision *models.SmPolicyDecision) *models.QosData {
```

Comment:
this api is not used, so is it added for future purpose?

Reply:
It is called from Datapath.go, in same changeset as 
defQosData := qos.GetDefaultQoSDataFromPolicyDecision(smPolicyDec)